### ISSUE TITLE
[Accessibility] Links under "Visualize and Simplify Platform Engineering" are having poor lighthouse accessibility score

### ISSUE THREAD

**Issue by OMGATE23** [layer5io/layer5#5415]
### Description
The Links under "Visualize and Simplify Platform Engineering" are not having alt tags or even any discernable names considering that they are Links.
This badly affects the accessibility score.

### Expected Behavior
The links should have proper aria-labels and alt tags

### Screenshots
The component in question
<img width="730" alt="Screenshot 2024-02-21 at 3 19 12 PM" src="https://github.com/layer5io/layer5/assets/90276220/cee9266e-a2e4-4be0-8c1c-31964cf0a52a">

What lighthouse says about this
<img width="619" alt="Screenshot 2024-02-21 at 3 23 00 PM" src="https://github.com/layer5io/layer5/assets/90276220/ba30d265-2a1a-40be-9086-238bf2aadf02">
<img width="627" alt="Screenshot 2024-02-21 at 3 24 08 PM" src="https://github.com/layer5io/layer5/assets/90276220/0084eead-a26b-4778-ae2f-77b17844b8b7">

### Environment:
- Host OS: MacOS
- Browser: Chrome

**I would like to work on this issue. I have earlier worked on similar issue [here](https://github.com/layer5io/layer5/issues/4266)**

---
<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/buttons/community.webp" height="22px" align="left" />Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

**Comment by OMGATE23** [comment]
**I would like to work on this issue**

**Comment by OMGATE23** [comment]
I further made the changes to check this issue if resolved will have how much impact and turns out its quite a bit

This is our previous accessibility score of 86
<img width="204" alt="Screenshot 2024-02-21 at 3 38 40 PM" src="https://github.com/layer5io/layer5/assets/90276220/60bb3cc7-e5ce-44eb-bfbb-857889496d6a">

By making these changes, we will reach the accessibility score on lighthouse to 91
<img width="208" alt="Screenshot 2024-02-21 at 3 37 47 PM" src="https://github.com/layer5io/layer5/assets/90276220/9dbe1e4c-081f-400a-8af6-db2f923bc3cd">

You can check the changes [here](https://github.com/OMGATE23/layer5/tree/features/twocollayout)

**Comment by vishalvivekm** [comment]
Thanks @OMGATE23
